packages/beacon-node/src/network/core/networkCore.ts

```diff
@@ -220,8 +219,11 @@ export class NetworkCore implements INetworkCore {
 
     // Network spec decides version changes based on clock fork, not head fork
     const forkCurrentSlot = config.getForkName(clock.currentSlot);
+
+    const boundary = {fork: forkCurrentSlot};
```

Comment:
```suggestion
    const boundary: SubscribeBoundary = {fork: forkCurrentSlot};
```

or could remove constant altogether